3531/0/about_dice_project.rb

```diff
@@ -0,0 +1,72 @@
+# rubocop:disable Layout/EndOfLine, Lint/UnneededCopDisableDirective, Lint/MissingCopEnableDirective
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#
+# class DiceSet
+#   code ...
+# end
+
+class DiceSet
+  attr_reader :values
+
+  def roll(number)
+    @values = []
+    number.times { @values << rand(1..6) }

```

Comment:
is it possible to do this in Array initialization?

Reply:
I did the right thing?